Gopkg.toml

```diff
@@ -1,17 +1,41 @@
+[[constraint]]
+  name = "github.com/operator-framework/operator-sdk"
+  # The version rule is used for a specific release and the master branch for in between releases.
+  branch = "master"
+  # version = "=v0.0.5"
+
+[[override]]
+  name = "github.com/json-iterator/go"
+  branch = "master"
+
+[[override]]
+  name = "github.com/docker/distribution"
+  branch = "master"
+
+[[override]]
+  name = "k8s.io/apiserver"
```

Comment:
Why are we importing this?

Reply:
@fabianvf any idea?

Reply:
None whatsoever, I just was flailing trying to get the dep ensure to work and adding this worked at the time. Implementation may have changes such that we no longer require it though, worth a try.

Reply:
I'm doing a quick audit of these imports. Some we don't need, and some are missing.